api-admin/src/main/java/bio/terra/pearl/api/admin/service/MailingListExtService.java

```diff
@@ -47,6 +49,43 @@ public List<MailingListContact> getAll(
     return mailingListContactService.findByPortalEnv(portalEnv.getId());
   }
 
+  @Transactional
+  public List<MailingListContact> create(
+      String portalShortcode,
+      EnvironmentName envName,
+      List<MailingListContact> contacts,
+      AdminUser user) {
+    Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
+    PortalEnvironment portalEnv =
+        portalEnvironmentService
+            .findOne(portal.getShortcode(), envName)
+            .orElseThrow(() -> new NotFoundException("Portal environment not found"));
+
+    List<MailingListContact> newContacts =
+        mailingListContactService.bulkCreate(portalEnv.getId(), contacts);
+
+    List<DataChangeRecord> changeRecords =
```

Comment:
I'm not sure we need to audit mailing list records, but if we do, we should just change MailingListService to extend DataAuditedService, and then you'll get a bulkCreate method for free :).  This will mean switching MailingListRecords from immutable to mutable, but that's fine since they're now audited